cdk/PgStacInfra.ts

```diff
@@ -33,7 +33,8 @@ export class PgStacInfra extends Stack {
           ? ec2.SubnetType.PUBLIC
           : ec2.SubnetType.PRIVATE_ISOLATED,
       },
-      allocatedStorage: 1024,
+      // set allocated stoarge to 20GB if stage is test, otherwise 50GB
+      allocatedStorage: stage === "test" ? 20 : 50,
```

Comment:
You could pass this in as an environment variable and then use github action environments? or just environment files

Reply:
Done !

Reply:
github action environment variable.